apps/api/src/campaign/dto/create-campaign.dto.ts

```diff
@@ -32,11 +32,6 @@ export class CreateCampaignDto {
   @IsString()
   essence: string
 
-  @ApiProperty()
-  @Expose()
-  @IsString()
-  bankHash: string
```

Comment:
Good catch, we don't need the payment reference in the input for campaign creation